src/commissaire_http/constants.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2016  Red Hat, Inc
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Constants specific to the HTTP related code.
+"""
+
+# JSONRPC error codes
+JSONRPC_ERRORS = {
+    'INVALID_JSON': -32700,
+    'INVALID_REQUEST': -32602,
+    'METHOD_NOT_FOUND': -32601,
+    'INVALID_PARAMETERS': -32602,
+    'INTERNAL_ERROR': -32603,
+}
```

Comment:
Moved them to https://github.com/projectatomic/commissaire/pull/14. Will update this PR with a fixup.